src/Altinn.App.Core/Features/Maskinporten/Models/JwtBearerToken.cs

```diff
@@ -0,0 +1,60 @@
+using System.ComponentModel;
+using Altinn.App.Core.Models;
+
+namespace Altinn.App.Core.Features.Maskinporten.Models;
+
+/// <summary>
+/// An object that contains a JWT <see cref="AccessToken"/> member
+/// </summary>
+public interface IAccessToken
```

Comment:
Do we need an interface for this? Seems kind of complicated to me to have 3 levels of types to really represent 1 object, which is the JWT token